Cargo.toml

```diff
@@ -97,4 +98,4 @@ debug = true
 debug = true
 
 [patch.crates-io]
-zcash_note_encryption = { version = "0.4", git = "https://github.com/QED-it/librustzcash.git", tag = "orchard_zsa_0.5.0_compatible" }
+zcash_note_encryption = { version = "0.4", git = "https://github.com/QED-it/librustzcash.git", branch = "zsa-bundle-in-v5-vector-serialization-with-burn-valid" }
```

Comment:
Let's have a PR with only these changes to librustzcash.

Reply:
use `zsa1-zebra` with changes to `zcash-note-encryption` to finalize.

Reply:
I copied the `components/zcash_note_encryption` folder from the `zsa1` branch to the `zsa1-zebra` branch, committed the changes there, and updated the `Cargo.toml` to use the updated branch to check if it works:

```toml
zcash_note_encryption = { version = "0.4", git = "https://github.com/QED-it/librustzcash.git", branch = "zsa1-zebra" }

It appears to compile successfully. Just to double-check, are you suggesting that I create a separate PR in the `orchard` repository to make `zsa1` reference `zcash_note_encryption` from the `zsa1-zebra` branch?